webapp/channels/src/components/admin_console/system_users/index.ts

```diff
@@ -40,6 +44,10 @@ function mapStateToProps(state: GlobalState) {
         tablePropertySortColumn: sortColumn,
         tablePropertySortIsDescending: sortIsDescending,
         tablePropertyPageSize: pageSize,
+        tablePropertyPageIndex: pageIndex,
+        tablePropertyDirection: direction,
+        tablePropertyFromId: fromId,
+        tablePropertyFromColumnValue: fromColumnValue,
```

Comment:
similar suggestions here to prefix them with suitable `cursor` term. as suggested before